### DISCUSSION TITLE
What is going on with the releases under different orgs?

### DISCUSSION THREAD

**Discussion by ian-cameron** [udotdevelopment/ATSPM#232] [General]
Somone sent me a link to a new repo I haven't seen before:
https://github.com/OpenSourceTransportation/Atspm

but it's a fork, and 58 commits behind this, other org I didn't know about-
[https://github.com/utahudot/udot-atspm](https://github.com/utahudot/udot-atspm)
 
 it has a long awaited release of 5.0 RC1.  

I'll guess you decided to reset history and cut over to a monster initial commit? @dloweAvenueConsultants  

But the same repo name with different casing? But why?
All your stars, forks, followers and collaborators are here.  
 
No READMEs of substance, so just asking here.  

Who is udotudot and who is OpenSourceTransportation?

Thank you!

**Comment by dloweAvenueConsultants** [comment]
Hi Ian,
There are 2 main reasons UDOT decided to go this direction. 

First 5.0 is a drastic change from 4.3. It was almost a complete re write. 5.0 will need to be  installed side by side and ported over from 4.3. Also because it is so drastic UDOT did not want to completely cut agencies off at the knees who are currently using it. They will mark this as deprecated but should agencies want or need to continue using this code base it will be available. The updated readme will be coming soon. 

Second the new repository is set up as an organization instead a single repository. As ATSPM grows it is being used in new and different ways. 5.0 can already ingest data from multiple different devices other than signal controllers. UDOT wanted to provide a place where potential modules for ATSPM and related software could reside together. UDOT can add teams to the organization to allow some different types of collaboration.

I am going to tag @ChesireFreeThrow from UDOT in case I missed anything.

**Comment by ian-cameron** [comment]
Okay thanks,  can you confirm which org is the one we should follow for 5.0?  (utahudot or OpenSourceTransportation)?  And UDOT still behind the new organization?  

We will be installing 5.0-rc1 and we'd still like to help test and contribute....  Is there any kind of documentation yet or architecture overview posted anywhere else I might be missing?

**Reply by dloweAvenueConsultants** [reply]
UDOT will still be the owners of the organization.

OpenSourceTransportation will be the source of truth. The other repository is what UDOT uses to code on and then they will push their changes to the OpenSourceTransportation organization when they are ready.

I will let UDOT speak to documentation and overview. I think they discussed presenting on it. One of the things they are hoping for is that the community will build installation scripts to share. Because it is container based these scripts are much easier to produce than previous installation requirements. Currently UDOT is building theirs in GCP using github actions. I believe they wanted to share a version of it that will allow others to install on GCP. As soon as I have sample settings I will get them added to the other repository. Also there is a tool called database installer in utilities that will build and apply migrations for your databases.

More info is coming. We are still early days. UDOT wanted to get the code out so that people can start familiarizing themselves with it.
